src/mcp/tools/crashlytics/list_top_issues.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { listTopIssues } from "../../../crashlytics/listTopIssues.js";
+
+export const list_top_issues = tool(
+    {
+        name:"list_top_issues",
+        description: "List the top issues happening in the application.",
+        inputSchema: z.object({
```

Comment:
Should this have `start_time` and `end_time` (optional) input fields? They can default to "7 days ago" and "now" but seems reasonable to make them configurable.

Reply:
Made the lookback period as an optionally configurable parameter defaulting it to 7.